### PR TITLE
Added /blockpvp /blockredstone and /blockedit

### PR DESCRIPTION
**Pull request by ghost**
Issues referenced: #2017, #2016 and #2015
Note: This is live on the server, but this version has formatting, logic and grammar improvements

### CODE REVIEW

**Review by mayokek — APPROVED** [review]
Looks good. All it needs is testing.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_blockedit.java [hunk 1]
```diff
@@ -24,7 +24,7 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
 
         if (args[0].equals("list"))
         {
-            msg("Blocked block edits players:");
+            msg("Block modification is disallowed for these players:");
```
Comment (mayokek) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_blockedit.java]: Maybe add 'the following' and remove 'these'?

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_blockpvp.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
+package me.totalfreedom.totalfreedommod.command;
```
Comment (LiamTownsley) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_blockpvp.java:1]: This is already a command implemented by @marcocorriero

**Review by LiamTownsley — CHANGES_REQUESTED** [review]
Currently I do not think that /blockredstone or /blockpvp are needed.

/blockredstone : If redstone is being abused ban the abuser.

/blockpvp : @marcocorriero already implemented this.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_blockpvp.java (hunk 2 above)
Comment (mayokek) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_blockpvp.java:1]: This PR was created before it was even implemented.

**Comment by savnithyt** [comment]
```
import org.bukkit.event.block.BlockPistonExtendEvent;
import org.bukkit.event.block.BlockPistonRetractEvent;
import org.bukkit.event.block.BlockRedstoneEvent;
```

Needs to be added in EventBlocker.java other than that the plugin works fine!

**Comment by savnithyt** [comment]
```
import org.bukkit.event.block.BlockPistonExtendEvent;
import org.bukkit.event.block.BlockPistonRetractEvent;
import org.bukkit.event.block.BlockRedstoneEvent;
```
^ That needs to be added in EventBlocker.java, other than that the code works fine.

**Review by Wild1145 — CHANGES_REQUESTED** [review]
This PR is currently not building for me. Can you look into this please?

**Comment by ghost** [comment]
@Wild1145 PR should build now.

**Comment by Wild1145** [comment]
@Commodore64x This is still failing the TravisCI Integration tests.

**Comment by JeromSar** [comment]
@Commodore64x As with the other branch. Please merge with `devel` to fix the Travis CI issue.

**Comment by ghost** [comment]
@JeromSar @Wild1145 Should work with travis now.

**Pull request closed without merging**